### PR TITLE
Demote from prod to unstable

### PR DESCRIPTION
**Pull request by prapti**
Demoting some of the prod tests that have been consistently failing in the last one week to unstable, with the idea that I'll take a look at them this week and promote back to prod if they pass by the end of this week.

### CODE REVIEW

File: e2e/cypress/integration/cloud/onboarding/cloud_sysadmin_onboarding_spec.js [hunk 1]
```diff
@@ -7,8 +7,6 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Stage: @prod
```
Comment (saturninoabril) [inline e2e/cypress/integration/cloud/onboarding/cloud_sysadmin_onboarding_spec.js:10]: Let's not include this. I'll try to submit fix for this.

File: e2e/cypress/integration/files_and_attachments/upload_files_spec.js [hunk 2]
```diff
@@ -7,7 +7,6 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Stage: @prod
```
Comment (saturninoabril) [inline e2e/cypress/integration/files_and_attachments/upload_files_spec.js:10]: Same here. I'll take a look.

**Review by saturninoabril — APPROVED** [review]
Thanks @prapti! I'll help take a look at cloud on boarding, file uploads and user status specs.

**Review by srkgupta — APPROVED** [review]
LGTM

**Pull request merged**